v2/pubsub-to-redis/src/main/java/com/google/cloud/teleport/v2/templates/PubSubToRedis.java

```diff
@@ -219,6 +219,8 @@ public interface PubSubToRedisOptions
     @TemplateParameter.Long(
         order = 8,
         optional = true,
+        parentName = "redisSinkType",
+        parentTriggerValues = {"HASH_SINK", "LOGGING_SINK"},
         description = "Hash key expiration time in sec (ttl)",
         helpText =
             "Key expiration time in sec (ttl, default for HASH_SINK is -1 i.e. no expiration)")
```

Comment:
Can you please also update the description and mention this parameter applies only to HASH_SINK and LOGGING_SINK sink types? It's true, based on the implementation, but this is not clear from the parameter description (it won't be visible in the UI once we add parentTriggerValues anyway, but we use the parameter description to auto-generate docs in some other places).

Reply:
Done